newIDE/app/src/Export/PreviewLauncher.flow.js

```diff
@@ -75,3 +57,20 @@ export type HotReloaderLog = {|
   kind: 'fatal' | 'error' | 'warning' | 'info',
   message: string,
 |};
+
+/** The functions that PreviewLauncher must expose on their class */
+export type PreviewLauncherInterface = {
```

Comment:
should the type be strict?

Reply:
It cannot be done without changing the LocalPreviewLauncher and BrowserS3PreviewLauncher being class components (the interface is not defined, you can use any method of the component) and I don't want to do this refactor in this PR to avoid possible bugs